.github/workflows/tests-rdbstorage.yml

```diff
@@ -0,0 +1,103 @@
+name: Tests (RDB storage)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+  schedule:
+    - cron: '0 23 * * SUN-THU'
+
+jobs:
+  tests-rdbstorage:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Do we need to run this test for every python version?

Reply:
I will run it also 3.9.

Reply:
Sounds good. I'd expect that to be a requirement.